tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -26,20 +33,15 @@ public void run() {
 
     @Override
     public void process(final Socket connection) {
-        try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
-
-            final var responseBody = "Hello world!";
+        try (InputStream inputStream = connection.getInputStream(); OutputStream outputStream = connection.getOutputStream()) {
```

Comment:
```suggestion
        try (InputStream inputStream = connection.getInputStream();
             OutputStream outputStream = connection.getOutputStream()) {
```

엔터를 넣어줘야 의미를 구분하기 쉬워집니다.